### ISSUE TITLE
[Entrega 3] Revertir ataque de Kane cuando usa su habilidad

### ISSUE THREAD

**Issue by sbvillalobosm** [RodrigoToroIcarte/IIC2113-2023-1#79]
Hola! Tengo una duda con respecto al caso en que Kane use su habilidad de que "que al iniciar su turno, antes de robar, el oponente está obligado a pegarse uno de daño" por lo que se vuelca la primera carta de su arsenal para que esta pase a su ringside. Como no es una carta de Kane la que genera este daño, sino que es su propia habilidad, ¿aunque vuelque una carta tipo Reversal, esta no debería tener la capacidad de revertir el ataque, cierto?

**Comment by RodrigoToroIcarte** [comment]
Exacto. El daño por la habilidad de Kane ocurre siempre y no hay forma de revertirlo (ni evitarlo).
